defaults/test_default.yaml

```diff
@@ -172,7 +172,7 @@ backup_bucket_region: ''  # use the same region as a cluster
 
 events_limit_in_email: 10
 
-scylla_bench_version: v0.1.6
+scylla_bench_version: v0.1.7
```

Comment:
so it must be updated to `v0.1.8`, @dkropachev ?

Reply:
Yap